prime-dotnet-webapi/Models/Enrollee.cs

```diff
@@ -219,6 +219,14 @@ public int DisplayId
             get => Id + DISPLAY_OFFSET;
         }
 
+        [NotMapped]
+        [Computed]
+        [JsonIgnore]
+        public string FullName
+        {
+            get => FirstName + " " + LastName;
```

Comment:
A have a (very slight) preference for interpolated strings when doing stuff like this. Could you change it to 
`get => $"{FirstName} {LastName}"`